scylladb/sphinx-scylladb-theme
# Add  customizable layout options per page

Closes https://github.com/scylladb/sphinx-scylladb-theme/issues/229#issue-1017848554

## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open http://127.0.0.1:5500/ with your favorite browser. The home page layout must look the same as https://sphinx-theme.scylladb.com/stable/. Furthermore, the internal page layout (e.g. Installation) must look the same as before https://sphinx-theme.scylladb.com/stable/getting-started/installation.html.

## Releasing this change

This PR introduces breaking changes. To ensure it doesn't affect other projects with the new theme, I'll release a new version increasing the minor number. Unfortunately, this will prevent projects from receiving the update automatically. 

After releasing ``1.1``, I'll submit PRs for the projects ``care-pet`` and ``sphinx-scylla-monitoring`` to install the latest version.
